paella-opencast/plugins/es.upv.paella.opencast.loader/05_initdelegate.js

```diff
@@ -43,9 +43,15 @@ function loadOpencastPaellaDCE(containerId) {
 			}
 			paella.load(containerId, {data:data});
 		},
-		function(){
-			// #DCE bypassing paella5 opencast auth, current DCE version is performed within getEpisode()
-			paella.messageBox.showError("Unable to load video " + paella.utils.parameters.get('id'));
+		function(jsonData){
+			// #DCE start specific DCE auth handling, formally in isHarvardDceAuth() (MATT-2212)
+			var dceAuthError = '';
+			if (jsonData && jsonData[ 'dce-auth-results'] && paella.opencast.isHarvardDceAuthRedirect(jsonData, dceAuthError)) {
```

Comment:
@lbjay you are right, thank you for seeing this. I talked to naomi and normally the player would never get to this point because the pin won't let the user back to this page if the user doesn't have access. The only case of having an error message here is if something is very wrong with the auth service or the path between auth and Mh. In which case, we probably shouldn't show the message to user, but leave the default "Unable to load..." and let the system alerts prompt us to fix the underlying issue. I'll update the code and push the change.